### PR TITLE
Updated presentation guidelines for 2020

### PR DESCRIPTION
**Pull request by borjasotomayor**
I've updated the presentation guidelines for 2020. The substance of what is required is pretty similar to last year, but the format is now different (pre-recorded presentations as opposed to live presentations).

### CODE REVIEW

File: website/content/project/presentation.md [hunk 1]
```diff
@@ -26,50 +34,71 @@ You must also discuss at least one thing that you would've done differently with
...
+## Submission Instructions
 
-The presentation will be worth 10 points of your final grade. 6 of these points will be determined by the instructor and TAs, and the remaining 4 points will be determined by a panel of guest judges. The  instructor/TAs will score your presentation (on a scale of 0 to 5) on three aspects: Clarity/structure (weighed 40%), demo (weighed 35%) and Q&A (weighed 25%). The guest judges will score your presentation only on clarity/structure (75%) and Q&A (25%).
+Submission instructions will be posted closer to the deadline, but we expect you will be able to upload your video through Panopto.
 
-Both the instructor/TAs and guest judges will use the following rubric, with discretion to add/subtract half a point.
+## Q&A
 
-### Clarity and structure
+During the Q&A meeting, you will be asked some questions based on your presentation (this is meant to emulate the Q&A that typically happens after a live presentation). As many team members as possible should attend the Q&A meeting, sp
```
Comment (dwahme) [inline website/content/project/presentation.md:57]: I think the end of this section got deleted

Comment (borjasotomayor) [reply website/content/project/presentation.md]: Oops, fixed.

File: website/content/project/presentation.md [hunk 2]
```diff
@@ -1,22 +1,30 @@
...
-{{% warning %}}
-**NOTE**: These are the requirements for the 2019 edition of CMSC 22000. They have not yet been updated for 2020!
-{{% /warning %}}
+The final component of this course will be a series of final presentations, where each team will present the work they have done throughout the quarter. This presentation will be divided into two components:
 
-The final component of this course will be a series of final presentations, where each team will present the work they have done throughout the quarter. The presentations will take place on Wednesday, June 12, from 1:30pm to 3:30pm, in Ryerson 277. Each team will have 15 minutes to present (with an additional 5 minutes for questions).
+- A pre-recorded presentation that must be submitted by **Tuesday, June 9, at 12pm CDT**.
+- A 5-10 minute Q&A session with the instructor and TAs during the "final exam" slot for the class: **Tuesday, June 9, from 4pm to 6pm CDT**
+
+Once all presentations have been submitted, they will be shared with the entire class, so that other teams can learn about the work that other teams have done (similar to how, in live presentations, you would be able to attend the presentations for all the teams).
```
Comment (firatciftci) [inline website/content/project/presentation.md:8]: Minor point about the language: "other" is repeated here, so maybe the first "other teams" should be changed to "teams" instead?

Comment (borjasotomayor) [reply website/content/project/presentation.md]: Sounds good, I've tweaked the wording.

**Pull request merged**